CONTRIBUTING.md

```diff
@@ -40,24 +39,24 @@ git push origin main
 ```
 
 ### 4. Create a New Branch And Open a Pull Request
-After you finish implementation, open forked repository. The source branch is your new branch, and the target branch is `OpenDevin/OpenDevin` `main` branch. Then PR should appears in [OpenDevin PRs](https://github.com/OpenDevin/OpenDevin/pulls).
+Create a new branch on your fork with your implementation. Open your forked repository and create a Pull Request which sets the source branch to your new branch, and the target branch to `OpenDevin/OpenDevin` `main` branch.
```

Comment:
For non-English speakers this sentence is hard to parse at first glance, not everyone is a major in Git-ology. :)
For many it is obvious, but not every developer uses Git daily.
Maybe a small screenshot can be a helpful illustration.

Reply:
That's great feedback. Let me see how I can make it more clear and not require a Git major haha.

Reply:
How does it look now?

Reply:
Reads great  now!